haystack/pipelines/base.py

```diff
@@ -182,6 +186,12 @@ def load_from_yaml(cls, path: Path, pipeline_name: Optional[str] = None, overwri
         """
 
         pipeline_config = cls._read_pipeline_config_from_yaml(path)
+        if pipeline_config["version"] != __version__:
+            logger.warning(
+                f"YAML version ({pipeline_config['version']}) does not match with Haystack version ({__version__}). "
+                "Issues may occurr during loading. "
```

Comment:
Can we add some advice here what could be done in such a case? E.g. In most cases loading a different version will work as expected. In case issues occur during loading, please check out our migration guide at https://haystack.deepset.ai/overview/migration or downgrade to haystack version `{__version__}`.

Reply:
Isn't the line just below a sufficient explanation in most cases though?